client/views/windows/connect.tpl

```diff
@@ -78,8 +78,11 @@
 		<div class="col-sm-3">
 			<label for="connect:password">Password</label>
 		</div>
-		<div class="col-sm-9">
+		<div class="col-sm-9 password-container">
 			<input class="input" id="connect:password" type="password" name="password" value="{{defaults.password}}">
+			<span class="reveal-password tooltipped tooltipped-n tooltipped-no-delay" aria-label="Show Password" data-alt-label="Hide Password">
```

Comment:
This is rather nitpicky, but in order to be consistent with similar elements like this, could you go for `Show password` and `Hide password` (i.e. drop the uppercase on `Password`)? Thanks!